resources/views/layouts/default/airports/show.blade.php

```diff
@@ -77,13 +77,13 @@
             @lang('flights.none')
             </div>
         @else
-            <table class="table table-striped table-condensed">
+            <table class="table table-striped">
                 <thead>
                 <tr>
                     <th class="text-left">@lang('airports.ident')</th>
-                    <th class="text-left">@lang('common.to')</th>
-                    <th>@lang('common.departure')</th>
-                    <th>@lang('common.arrival')</th>
+		    <th class="text-left">@lang('airports.arrival')</th>
+                    <th>@lang('flights.dep')</th>
+		    <th>@lang('flights.arr')</th>
```

Comment:
Looks good! Only comment is to align the spacing here (only spaces, no tabs). Good catch on the language